docs/medical-api/api-reference/patient/bulk-create-patient.mdx

```diff
@@ -0,0 +1,250 @@
+respective status will be stored so it can be reviewed in the
+[result CSV file](/medical-api/handling-data/webhooks#result-csv-file) (it won't fail the whole
+job at this point).
+
+### Uploading the CSV
+
+The URL is pre-signed to your account and has a duration of 10 minutes. It should only be used
+once. Use it to send a `PUT` request like so:
+
+```shell
+curl --location --request PUT '<UPLOAD-URL>' \
+--header 'Content-Type: text/csv' \
+--header 'x-api-key: <YOUR-API-KEY>' \
+--data-binary '@/local-to-your/file.csv'
+```
+
+#### Webhook with the result
+
+Once the CSV file is uploaded, the Metriport API will start processing it and will send
+[webhook messages](/medical-api/handling-data/webhooks#bulk-patient-create) to your App
+indicating the status of the processing, including the resulting CSV file containing the
+newly created Metriport IDs for the successfully created Patients.
+
+#### Bulk Patient Create Status
+
+During the processing of a bulk patient create, the job can change status following the logic
+below:
+
+<img src="/images/bulk-import-status.png" />
+
+```mermaid
+---
+config:
+  theme: mc
+  look: handDrawn
+  layout: dagre
+---
+flowchart LR
+    A["waiting"] --> B["processing"] & C["failed"] & D["expired"]
+    B --> E["completed"]
+```
+
+- `waiting`: initial state, waiting for the CSV file;
+- `processing`: the CSV file has been uploaded and is being processed;
+- `completed`: the bulk import job is completed;
+- `failed`: the bulk import job failed (likely due to an uploaded file with invalid format);
+- `expired`: no file has been uploaded and the upload URL expired.
+
+## Query Params
+
+<ParamField query="facilityId" type="string">
+  The ID of the Facility where the Patients receive care.
+  All patients in a single job/CSV will be associated with
+  the same Facility. It can be omitted if there's only one
+  Facility in your account.
+</ParamField>
+
+<ParamField query="dryRun" type="boolean">
+  When set to `true`, no Patients will be created, and this
+  job is only used to validate the CSV file and provide
+  feedback about each Patient demographics. Defaults to
+  `false`.
+</ParamField>
+
+## Body
+
+No body is accepted/processed.
+
+## Response
+
+<ResponseField name="jobId" type="string" required>
+  The ID that represents this bulk patient create job.
+</ResponseField>
+
+<ResponseField name="status" type="string" required>
+  The status of the job. Always `waiting`. See all the
+  possible values [here](#bulk-patient-create-status).
+</ResponseField>
+
+<ResponseField name="uploadUrl" type="string" required>
+  The URL to upload the CSV file containing the Patient
+  demographics to be used to create new Patients. It's valid
+  for 10 minutes and should only be used once. See more
+  details [here](#uploading-the-csv)
+</ResponseField>
+
+<ResponseExample>
+
+Coming soon.
+
+</ResponseExample>
+
+```json
+{
+  "jobId": "00000000-00000000-00000000-00000000",
```

Comment:
we use `requestId` in the Start Document Query endpoint - let's use that here to be consistent?

Reply:
done